src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -254,7 +289,13 @@ public SectionDetailsBundle getSectionForCourse(String section, String courseId)
         return sectionDetails;
     }
     
-    public List<SectionDetailsBundle> getSectionsForCourse(CourseAttributes course, CourseDetailsBundle cdd) {
+    /**
+     * Returns a list of {@link SectionDetailsBundle section details} for a given course using course attributes and course details bundle.
```

Comment:
This line seems a bit too long. I have to scroll horizontal to read it in GitHub diff view.